frontend/packages/topology/integration-tests/support/step-definitions/export-application/export-of-application.ts

```diff
@@ -0,0 +1,93 @@
+import { When, Then, Given } from 'cypress-cucumber-preprocessor/steps';
+import { devNavigationMenu } from '@console/dev-console/integration-tests/support/constants/global';
+import { createGitWorkload } from '@console/dev-console/integration-tests/support/pages';
+import {
+  app,
+  navigateTo,
+  projectNameSpace,
+} from '@console/dev-console/integration-tests/support/pages/app';
+import { exportApplication, exportModalButton } from '../../page-objects/export-applications-po';
+import { topologyPO } from '../../page-objects/topology-po';
+import { exportOfApplication } from '../../pages/export-application/export-applications';
+import { topologyHelper, topologyPage } from '../../pages/topology';
+
+Given('user has created a deployment workload {string}', (workloadName: string) => {
+  navigateTo(devNavigationMenu.Add);
+  createGitWorkload(
+    'https://github.com/sclorg/nodejs-ex.git',
+    workloadName,
+    'Deployment',
+    'nodejs-ex-git-app',
+  );
+});
+
+When('user navigates to Topology page', () => {
+  navigateTo(devNavigationMenu.Topology);
+});
+
+When('user clicks on Export Application button', () => {
+  exportOfApplication.exportApplicationFresh();
+});
+
+Then('user can see a toast message saying {string}', (message: string) => {
+  cy.get(exportApplication.infoTip).contains(message);
+});
+
+Then('user can see primer job created in topology', () => {
+  cy.wait(5000);
+  topologyPage.verifyWorkloadInTopologyPage('primer');
```

Comment:
I'm afraid if this 8s wait (and the other three waits in this PR) makes our tests flaky in the future.

Isn't it common in cypress to run the verification with a timeout? Aka the most cypress commands like `cy.get` waits until the element is found.

This is not a requirement to get approval, but I want bring the idea here to add an optional `options` parameter to `verifyWorkloadInTopologyPage` instead and use a longer timeout in the primer case where I agree that it will take longer than the default 3 seconds.

Wdyt? Makes this sense for all waits here?

```suggestion
  topologyPage.verifyWorkloadInTopologyPage('primer', { timeout: 10000 });
```

And in `topology-helper-page.ts`:

```diff
-  verifyWorkloadInTopologyPage: (appName: string) => {
+  verifyWorkloadInTopologyPage: (appName: string, options?: { timeout: number }) => {
    topologyHelper.search(appName);
    // eslint-disable-next-line promise/catch-or-return
    cy.get('body').then(($body) => {
      if (
        $body.find('[data-test-id="topology-switcher-view"][aria-label="Graph view"]').length !== 0
      ) {
        cy.get(topologyPO.list.switcher).click();
        cy.log('user is switching to graph view in topology page');
      } else {
        cy.log('You are on Topology page - Graph view');
      }
    });
-    cy.get(topologyPO.highlightNode).should('be.visible');
+    cy.get(topologyPO.highlightNode, options).should('be.visible');
    app.waitForDocumentLoad();
  },
```

Disclaimer: I don't test this approach, but would be happy if you can check if this makes our timeouts more "flexible". Having a longer timeout instead of a `wait` call will run the tests faster if the exports runs quicker, and its doesn't fail because we can also wait a little bit longer then.

Reply:
Tested passed locally in 187 seconds (on a local cluster/CRC, local dev console) :+1: 

![image](https://user-images.githubusercontent.com/139310/143020399-4ad0cea8-718b-4a3b-8223-7c2f1f2ec5c3.png)

Reply:
Replaced wait with the timeout, it's working fine on my local